enterprise/server/remote_execution/action_merger/action_merger.go

```diff
@@ -22,12 +22,21 @@ const (
 	// set to twice the length of `remote_execution.lease_duration` to give a
 	// short grace period in the event of missed leases.
 	DefaultClaimedExecutionTTL = 20 * time.Second
+
+	// Redis Hash keys for storing the canonical execution ID and the pending
+	// execution count.
+	executionIDKey    = "execution-id"
+	executionCountKey = "execution-count"
 )
 
 var (
 	enableActionMerging = flag.Bool("remote_execution.enable_action_merging", true, "If enabled, identical actions being executed concurrently are merged into a single execution.")
+	hedgedActionCount   = flag.Int("remote_execution.action_merging_hedge_count", 0, "When action merging is enabled, this flag controls how many additional, 'hedged' attempts an action is run in the background. Note that even hedged actions are run at most once per execution request.")
 )
 
+// Returns the redis key pointing to the hash storing action merging state. The
+// value stored here is a hash containing the canonical (first-submitted)
+// execution ID and the count of executions run for this action (for hedging).
```

Comment:
You will want to change the name of the key to avoid rollout compatibility issues.

Reply:
Good call, added a version to both keys for future ease of use